h2o-core/src/test/java/water/rapids/ast/prims/PermutationFeatureImportanceTest.java

```diff
@@ -0,0 +1,444 @@
+package water.rapids.ast.prims;
+import hex.Model;
+import hex.glm.GLM;
+import hex.glm.GLMModel;
+import hex.tree.gbm.GBMModel;
+import hex.tree.gbm.GBM;
+import org.junit.Assert;
+import org.junit.Test;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.fvec.NFSFileVec;
+import water.fvec.TestFrameBuilder;
+import water.fvec.Vec;
+import water.parser.ParseDataset;
+import water.rapids.*;
+import water.rapids.vals.ValFrame;
+import water.util.*;
+import org.junit.BeforeClass;
+
+import java.util.Arrays;
+
+import static hex.genmodel.utils.DistributionFamily.gaussian;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class PermutationFeatureImportanceTest extends TestUtil {
```

Comment:
The tests here are usually not completed. Printing the feature importance at the end of the test is not correct test case. More work is needed here.

I miss a test where the feature importance from original model (for example, GLM model) is compared with permutation feature importance.

Reply:
What is the difference between PermutationFeatureImportanceTest.java and PermutationVarImpTest.java file? Keep only one or explain their purpose, please.

Reply:
Will keep only PermutationVarImpTest.java